langchain4j-bedrock/src/main/java/dev/langchain4j/model/bedrock/BedrockCohereEmbeddingModel.java

```diff
@@ -0,0 +1,78 @@
+package dev.langchain4j.model.bedrock;
+
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.model.bedrock.internal.AbstractBedrockEmbeddingModel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Bedrock Cohere Embedding Model with support for both versions:
+ * {@code cohere.embed-english-v3} and it's multilingual version {@code cohere.embed-multilingual-v3}
+ * <br>
+ * See more details <a href="https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters-embed.html">here</a>.
+ */
+@SuperBuilder
+@Getter
+public class BedrockCohereEmbeddingModel extends AbstractBedrockEmbeddingModel<BedrockCohereEmbeddingResponse> {
+
+    @Builder.Default
+    private final String model = Types.CohereEmbedEnglishTextV3.getValue();
```

Comment:
please do not set the default model, let user explicitly set it

Reply:
I'm fine with this.
I've used BedrockTitanEmbeddingModel.java written by Pascal Vanterpote and Jonny Coddington.
There model is set at least to default, whenever you have options to choose from.

I'll remove it if you insist, even after discussion.

Reply:
Please remove, we are moving away from default mdoel names, as they become outdated pretty fast

Reply:
please address this comment